src/modules/android.ts

```diff
@@ -39,6 +39,7 @@ export type BuyItemByType = (
   type: string,
   skus: Sku[],
   purchaseToken: string | undefined,
+  externalTransactionID: string | undefined,
```

Comment:
If this is only android feature, could you differentiate props with additional suffix like
`externalTransactionIdAndroid`